agentBuilderWorkspace.json

```diff
@@ -1 +1 @@
-{"nodes":[{"width":98,"height":36,"id":"1","type":"rootNode","position":{"x":0,"y":0},"deletable":false,"data":{"nodeTypeName":"Root Node","nodeTypeDescription":"This is the Root Node, from which the rest of the graph is built.","label":"Root Node"},"positionAbsolute":{"x":0,"y":0}},{"width":167,"height":52,"id":"2","type":"sequenceNode","position":{"x":0,"y":100},"deletable":false,"data":{"isConnectableTarget":false,"nodeTypeName":"Sequence Node","nodeTypeDescription":"This is the starting point for your bot's Behavior Tree logic.","label":"Top Level Sequence Node"},"positionAbsolute":{"x":0,"y":100}}],"edges":[{"animated":true,"id":"e1-2","source":"1","target":"2","deletable":false}],"viewport":{"x":0,"y":0,"zoom":2}}
\ No newline at end of file
+{"nodes":[{"width":98,"height":36,"id":"1","type":"rootNode","position":{"x":350,"y":75},"deletable":false,"data":{"nodeTypeName":"Root Node","nodeTypeDescription":"This is the Root Node, from which the rest of the graph is built.","label":"Root Node"},"positionAbsolute":{"x":350,"y":75}},{"width":167,"height":52,"id":"2","type":"sequenceNode","position":{"x":325,"y":175},"deletable":false,"data":{"isConnectableTarget":false,"nodeTypeName":"Sequence Node","nodeTypeDescription":"This is the starting point for your bot's Behavior Tree logic.","label":"Top Level Sequence Node"},"positionAbsolute":{"x":325,"y":75}}],"edges":[{"animated":true,"id":"e1-2","source":"1","target":"2","deletable":false}],"viewport":{"x":0,"y":0,"zoom":2}}
```

Comment:
what are these positions relative to ?

in other words, if I have a HUGE screen, would these still be centered ?